src/handleScroll.ts

```diff
@@ -1,10 +1,12 @@
-import { Axis } from './types';
+import {Axis} from './types';
 
 const alwaysContainsScroll = (node: HTMLElement): boolean =>
   // textarea will always _contain_ scroll inside self. It only can be hidden
   node.tagName === 'TEXTAREA';
 
 const elementCanBeScrolled = (node: HTMLElement, overflow: 'overflowX' | 'overflowY'): boolean => {
+  if (!(node instanceof HTMLElement)) return false;
```

Comment:
👍  simple and sound

Reply:
should it give a warning? in case someone passed the wrong variable, really hard to debug the situation w/o info (could be wrong)

Reply:
this is internal-facing code, that _someone_ can be only the script itself.

Reply:
@theKashey Thanks for the comment! Feel free to merge if you're happy with the change. Otherwise, let me know if there are any tweaks you'd like me to make.